.github/workflows/enos-run.yml

```diff
@@ -222,6 +222,7 @@ jobs:
           ENOS_VAR_test_email: ${{ secrets.SERVICE_USER_EMAIL }}
           ENOS_VAR_boundary_docker_image_name: ${{ inputs.docker-image-name }}
           ENOS_VAR_boundary_docker_image_file: ./support/boundary_docker_image.tar
+          ENOS_DEBUG_DATA_ROOT_DIR: ./support/debug-data
```

Comment:
How does enos create the data in this directory. Will this data get overwritten by the debug data in the following step?

Reply:
The provider automatically will download the log files on failure. The log files will follow the pattern: `boundary_<host ip>.log`. If the scenario is run again the file will have a different host ip address so it will not be overwritten.

Reply:
Right now, we're adding this variable **for each scenario** in these steps
- Output Terraform version info
- Run Enos scenario

Are there any conflicts there?

Reply:
Not sure I quite understand you're comment. Clarifying a bit more the mechanism of how this works, may answer your question...

1. Application logs are only downloaded during a terraform plan or apply.
2. The feature is part of the enos Terraform provider, so logs are only downloaded if an enos resource fails, specifically (in the case of boundary), `enos_boundary_start` and any `enos_remote_exec`.
3. If during re-run, the same resource fails (or any other that accesses the same host),  the log files will be downloaded once again and will overwrite the existing downloaded log. In this case it is okay since presumably the new log file will fully contain the contents of the previously downloaded log file and any new information since the last time the logs were captured.

Reply:
Ultimately, my question is why this new environment variable is being added to two spots and if it's necessary, how it works. The PR is adding this environment variable to two steps:
- `Output Terraform version info` which does an `enos scenario check`
- `Run Enos scenario` which does an `enos scenario run`
- Then, if `Run Enos Scenario` fails, upload the logs

So, do the logs from the `Output Terraform version info` provide anything? If that step fails, the logs don't get uploaded (logs only get uploaded if `run` fails). If that step passes but `run` fails, does the information from `output terraform version info` provide any additional information?

Happy to jump on a call to clarify further.

Reply:
Ah, okay, now I understand your comment. Yeah there's no need to add the env var to the output step. It should only be added to the two steps where you are actually running the scenario. I will update. Good catch.